src/AdScript.js

```diff
@@ -4,7 +4,7 @@ const AdScript = () => (
   <script
     async
     src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-1368599666656039"
-    crossorigin="anonymous"
+    crossOrigin="anonymous"
```

Comment:
what will this line do?

Reply:
in React, we follow the camel-case naming convention, so for example, we add `classname` attr in react like this `className`. 

Other than that `cross-origin` is an attribute that is required for reloading scripts from different origins, it tells the script host to give access to another origin that is requesting the file.